packages/ui/v2/core/layouts/SettingsLayout.tsx

```diff
@@ -1,5 +1,6 @@
 import { Collapsible, CollapsibleContent, CollapsibleTrigger } from "@radix-ui/react-collapsible";
```

Comment:
It was my understanding that I was handling this part of teams. I have #4347 open right now waiting for this PR. I see how you've done the routing. What's the best way to continue without causing too much merge conflicts?

Reply:
I merged your branch [`v2/settings-shell`](https://github.com/calcom/cal.com/tree/v2/settings-shell) into this branch to already connect the shell with my implemented views. I asked @zomars, he said it's best to open up a PR into `v2/settings-shell` branch. I think once your branch is merged, this should automatically set the PR to `main`